haiku-apps/bescreencapture/bescreencapture-2.8.0.recipe

```diff
@@ -5,40 +5,30 @@ to any media format supported in Haiku.
 BeScreenCapture can record either the entire screen, or just a section you \
 select."
 HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
-COPYRIGHT="2014-2022 Stefano Ceccherini"
+COPYRIGHT="2014-2023 Stefano Ceccherini"
 LICENSE="BSD (3-clause)
 	MIT"
-REVISION="1"
+REVISION="3"
```

Comment:
As this is a new release, the REVISION should be reset back to "1".

Reply:
That's strange: I did test this on beta4 and It worked correctly. Is this master?

Reply:
Indeed, I'm on a nightly Haiku.
If all fails, you could provide an HTML version in the repo and only have a short ReadMe.md that's used on the github page, pointing to the actual HTML file. To avoid keeping two versions in sync.

Reply:
That's not a huge issue, I could do a 2.8.1 if that's a problem. Tomorrow I'll try with a nightly to see what's wrong with mmd.

Reply:
That's not a huge issue, I could do a 2.8.1 if that's a problem. Tomorrow I'll try with a nightly to see what's wrong with mmd.

Reply:
I was able to build correctly on master, too, but I found the problem:
mtools also provides the mmd command, so it clashes with the one I'm using from MultiMarkdown.
Any hint on how to fix this in the recipe ?